apps/src/code-studio/initApp/project.js

```diff
@@ -1164,7 +1164,7 @@ var projects = (module.exports = {
             firstSaveTimestamp = response.timestamp;
           }
           currentSourceVersionId = response.versionId;
-          replaceCurrentSourceVersion = true;
+          replaceCurrentSourceVersion = !forceNewVersion;
```

Comment:
This theoretically affects all projects but the only other place we pass `forceNewVersion = true` into this function is  in the VersionHistory dialog when [resetting the project](https://github.com/code-dot-org/code-dot-org/blob/1ca773e09737423c4ab7fc8cebdfd4dd4b6aac53/apps/src/templates/VersionHistory.jsx#L132). When resetting the project, the page is reloaded so `replaceCurrentSourceVersion` goes back to its default anyway.